packages/eas-cli/src/metadata/config.ts

```diff
@@ -12,6 +19,74 @@ export interface MetadataConfig {
   apple?: AppleMetadata;
 }
 
+/**
+ * Load the store configuration from a metadata context.
+ * This can load `.json` and `.js` config files, using `require`.
+ * It throws MetadataValidationErrors when the file doesn't exist, or contains errors.
+ * The user is prompted to try anyway when errors are found.
+ */
+export async function loadConfigAsync({
+  projectDir,
+  metadataPath,
+}: Pick<MetadataContext, 'projectDir' | 'metadataPath'>): Promise<MetadataConfig> {
+  const configFile = path.join(projectDir, metadataPath);
+  if (!(await fs.pathExists(configFile))) {
+    throw new MetadataValidationError(`Metadata store config file not found: "${configFile}"`);
+  }
+
+  const configData = require(configFile);
+  const { valid, errors: validationErrors } = validateConfig(configData);
+
+  if (!valid) {
+    const error = new MetadataValidationError(
+      `Metadata store config errors found`,
+      validationErrors
+    );
+
+    logMetadataValidationError(error);
+    Log.newLine();
+    Log.warn(
+      'Without further updates, the current store configuration may fail to be synchronized with the App Store or pass App Store review.'
+    );
+
+    if (await confirmAsync({ message: 'Do you still want to push the store configuration?' })) {
```

Comment:
Maybe it's better to warn the user that if they don't accept, EAS CLI will fail.

Reply:
I think it's clear enough that the CLI will exit with an error. It may be a bit repetitive as is though (it just exits with the original error that was printed before). But happy to improve it!

### When prompting

<img width="2560" alt="Screenshot 2022-08-12 at 16 33 34 (3)" src="https://user-images.githubusercontent.com/1203991/184377017-bf078e66-1317-4f62-a75e-203f8022323e.png">


### When answering no

![Screenshot 2022-08-12 at 16 34 35](https://user-images.githubusercontent.com/1203991/184377146-3e64daec-9004-42ae-bac6-6f2a677258c6.png)